pkg/reconciler/pipeline/dag/dagv1beta1_test.go

```diff
@@ -391,18 +392,39 @@ func TestBuild_Invalid_v1beta1(t *testing.T) {
 		name string
 		spec v1beta1.PipelineSpec
 	}{{
+		// a
+		// |
+		// a ("a" depends on resource from "a")
 		name: "self-link-from",
 		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{selfLinkFrom}},
 	}, {
+		// a
+		// |
+		// a ("a" runAfter "a")
 		name: "self-link-after",
 		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{selfLinkAfter}},
 	}, {
+		// a (also "a" depends on resource from "z")
+		// |
+		// x ("x" depends on resource from "a")
+		// |
+		// z ("z" depends on resource from "x")
 		name: "cycle-from",
 		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{xDependsOnA, zDependsOnX, aDependsOnZ}},
 	}, {
+		// a (also "a" runAfter "z")
+		// |
+		// x ("x" runAfter "a")
+		// |
+		// z ("z" runAfter "x")
 		name: "cycle-runAfter",
 		spec: v1beta1.PipelineSpec{Tasks: []v1beta1.PipelineTask{xAfterA, zAfterX, aAfterZ}},
 	}, {
+		// a (also "a" depends on resource from "z")
+		// |
+		// x ("x" depends on resource from "a")
+		// |
+		// z ("z" runAfter "x")
```

Comment:
Can we include these cycles in these comments too (and the one on the new test)?

e.g. something like this (or better! haha it's hard to show the direction):

```
 → a
|  ↓   
|  x
 \ ↓ 
   z
```

Reply:
Yeah I tried adding them but looked very weird. Wish I could apply my artistic skills here 🤣

Reply:
```
		// -> a
		// |  |
		// |  v
		// -- a ("a" depends on resource from "a")
```

🤔

Reply:
i think anything is fine, even if it doesnt look great, just something to make it a complete representation of the graph in question